### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by edsko**

### CODE REVIEW

File: src/exec/Main.hs [hunk 1]
```diff
@@ -324,15 +322,22 @@ runShelleyServer
   -> IO ()
 runShelleyServer addr _ ctable rversions = do
   addrInfo <- resolveAddress addr
+  peerStatesVar <- newPeerStatesVar
   withServer
+    nullTracer
```
Comment (karknu) [inline src/exec/Main.hs:327]: I think that all these `nullTracers` will be problematic. It looks like there will be no visibility into any network issue on the Shelley side of the proxy.

Comment (edsko) [reply src/exec/Main.hs:327]: I don't disagree but they were already present, independent of this PR.

**Comment by edsko** [comment]
As a sanity check, confirms that cardano-node can still sync from a local proxy.

**Comment by edsko** [comment]
Just noticed that I need to add a constraint on `katip` in the stack file. Will push.

File: src/exec/Main.hs (hunk 1 above)
Comment (karknu) [reply src/exec/Main.hs:327]: Those `nullTracer`s can be addressed in a later PR.

**Review by karknu — APPROVED** [review]
Looks good from a networking view.

File: src/exec/Main.hs [hunk 2]
```diff
@@ -17,7 +17,7 @@ import Control.Monad.Trans.Except (runExceptT)
 import Control.Tracer (Tracer (..), contramap, nullTracer, traceWith)
 import Data.Functor.Contravariant (Op (..))
 import Data.List (intercalate)
-import qualified Data.Reflection as Reflection (give, given)
+import qualified Data.Reflection as Reflection (given)
```
Comment (mrBliss) [inline src/exec/Main.hs:20]: Why is `reflection` still needed?

**Comment by edsko** [comment]
Thanks for the review @karknu !

**Review by mrBliss — APPROVED** [review]

File: src/exec/Main.hs (hunk 2 above)
Comment (edsko) [reply src/exec/Main.hs:20]: Because this talks to `cardano-sl` which uses reflection basically _everywhere_.

**Pull request merged**